docs/content/howtos/simulate-ransomware.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to simulate a ransomware attack"
+draft: false
+pre: '<i class="fa fa-lock"></i> '
+tags: ["howtos", "simulate ransomware"]
+---
+
+Follow these steps to successfully simulate a ransomware attack on your
+network.
+
+1. Prepare your environment for a ransomware simulation
+
+    The Infection Monkey will only encrypt files that are allowed by you.
+    Prepare your network by adding a "ransomware target" directory to each
+    machine in your environment. This can be done using a configuration
+    management tool, such as
+    [Ansible](https://docs.ansible.com/ansible/latest/user_guide/) or
+    [PsExec](https://theitbros.com/using-psexec-to-run-commands-remotely/), or
+    a Windows GPO.
+
+1. Configure the simulation
+
+    In the configuration, specify which directory the Infection Monkey should
```

Comment:
Which configuration?

Reply:
There's a screenshot of the configuration page right below this and the tutorial for ransomware describes how one can reach the configuration page. Since this is a how-to guide, I don't think we need to hold the user's hand through every little thing.